sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  requested_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  active_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  application_offering_change_request_status sims.application_offering_change_request_status_types NOT NULL,
+  assessed_date TIMESTAMP WITH TIME ZONE,
+  assessed_by INT REFERENCES sims.users(id),
+  student_action_date TIMESTAMP WITH TIME ZONE,
+  reason VARCHAR(500) NOT NULL,
+  assessed_note_id INT REFERENCES sims.notes(id),
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id),
+  modifier INT NULL DEFAULT NULL REFERENCES sims.users(id)
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.application_offering_change_requests IS 'Represents the list of application specific offering change requests, which is requested by institution.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.id IS 'Auto-generated sequential primary key column.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.application_id IS 'Foreign key reference to the applications table. Application on which the application specific offering change was requested.';
```

Comment:
May be, we can avoid technical terms like Foreign key in the comments as it is implicit from code.

Reply:
we are using it in other places, i was trying to follow this same,
![image](https://github.com/bcgov/SIMS/assets/77353155/24007560-6e4c-4cd0-a7c8-ccfae5c4cb66)

Reply:
If we have it already, may be we must stop using it IMO if other devs are onboard with the same idea. 

Btw. it is not a blocker.

Reply:
I am ok with not using the foreign key term explicitly.